### PR TITLE
Feature/sdl 0255 enhance body information vehicle data

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Implements https://github.com/smartdevicelink/rpc_spec/issues/207

This PR is **[ready]** for review.

### Summary
Enhance BodyInformation vehicle data

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by KhrystynaDubovyk** [comment]
@atiwari9 please review this PR

**Review by atiwari9 — APPROVED** [review]
Approved

**Comment by LitvinenkoIra** [comment]
@JackLivio This PR is ready for Livio review. Thank you!

File: MOBILE_API.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.0.0" minVersion="1.0" date="2020-11-04">
+<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2020-11-04">
```
Comment (iCollin) [inline MOBILE_API.xml]: can you please merge in develop to this branch and ensure the date here is April 2021? 

Thank you, looks good otherwise.

Comment (LitvinenkoIra) [reply MOBILE_API.xml]: @iCollin Thank you. Done in f5c56f3e4cc6c6735fb26ca239a895be4393a969

**Review by iCollin — APPROVED** [review]

**Pull request merged**